features/graph/service/src/main/java/org/opennms/netmgt/graph/service/DefaultGraphService.java

```diff
@@ -133,10 +133,12 @@ public void onBind(GraphContainerProvider graphContainerProvider, Map<String, St
     }
 
     public void onUnbind(GraphContainerProvider graphContainerProvider, Map<String, String> props) {
-        graphContainerProviders.remove(graphContainerProvider);
-        final ServiceRegistration<GraphContainerProviderRegistration> serviceRegistration = serviceRegistrationMap.remove(graphContainerProvider);
-        if (serviceRegistration != null) {
-            serviceRegistration.unregister();
+        if(graphContainerProvider != null) {
```

Comment:
Could be simplified by returning immediately if null - would save having to indent the whole block.